openpype/hosts/hiero/api/otio/hiero_export.py

```diff
@@ -10,23 +10,23 @@
 import hiero.core
 import hiero.ui
 
-self = sys.modules[__name__]
-self.track_types = {
-    hiero.core.VideoTrack: otio.schema.TrackKind.Video,
-    hiero.core.AudioTrack: otio.schema.TrackKind.Audio
-}
-self.project_fps = None
-self.marker_color_map = {
-    "magenta": otio.schema.MarkerColor.MAGENTA,
-    "red": otio.schema.MarkerColor.RED,
-    "yellow": otio.schema.MarkerColor.YELLOW,
-    "green": otio.schema.MarkerColor.GREEN,
-    "cyan": otio.schema.MarkerColor.CYAN,
-    "blue": otio.schema.MarkerColor.BLUE,
-}
-self.timeline = None
-self.include_tags = True
 
+class CTX:
+    track_types = {
+        hiero.core.VideoTrack: otio.schema.TrackKind.Video,
+        hiero.core.AudioTrack: otio.schema.TrackKind.Audio
+    }
+    project_fps = None
+    marker_color_map = {
+        "magenta": otio.schema.MarkerColor.MAGENTA,
+        "red": otio.schema.MarkerColor.RED,
+        "yellow": otio.schema.MarkerColor.YELLOW,
+        "green": otio.schema.MarkerColor.GREEN,
+        "cyan": otio.schema.MarkerColor.CYAN,
+        "blue": otio.schema.MarkerColor.BLUE,
+    }
+    timeline = None
+    include_tags = True
 
```

Comment:
I would make `track_types` and `marker_color_map` as constants. They're not changing.
```suggestion
TRACK_TYPE_MAP = {
    hiero.core.VideoTrack: otio.schema.TrackKind.Video,
    hiero.core.AudioTrack: otio.schema.TrackKind.Audio
}
MARKER_COLOR_MAP = {
    "magenta": otio.schema.MarkerColor.MAGENTA,
    "red": otio.schema.MarkerColor.RED,
    "yellow": otio.schema.MarkerColor.YELLOW,
    "green": otio.schema.MarkerColor.GREEN,
    "cyan": otio.schema.MarkerColor.CYAN,
    "blue": otio.schema.MarkerColor.BLUE,
}
    
class CTX:
    project_fps = None
    timeline = None
    include_tags = True

```